shared/src/bundles/users/types/user-get-user-profile-response-dto.type.ts

```diff
@@ -0,0 +1,15 @@
+type UserProfileResponseDto = {
+    email: string;
+    id: string | null;
+    createdAt: string | null;
+    updatedAt: string | null;
+    userId: string | null;
+    firstName: string | null;
+    lastName: string | null;
+    sex: string | null;
+    dateOfBirth: string | null;
+    language: string | null;
+    currency: string | null;
+};
```

Comment:
```suggestion
type UserProfileResponseDto = {
    email: string;
    id: string;
    createdAt?: string;
};
```

Reply:
make other field not required

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/300/commits/b86fdf028d4bb36d00ca5b87354b44cbb450e24e](url)